src/examples/file-upload/uploaded/index.njk

```diff
@@ -5,15 +5,21 @@ layout: layout-example.njk
 {% from "govuk/components/summary-list/macro.njk" import govukSummaryList %}
 {% from "govuk/components/tag/macro.njk" import govukTag %}
 
-<h1 class="govuk-heading-xl">Upload receipt</h1>
+<h1 class="govuk-heading-xl">Uploaded receipt</h1>
```

Comment:
@Jon-Rowe-HMRC I think this would ready better as 'Receipt uploaded'. What do you think?